### PR TITLE
Clarify details around review approvals before merging

### PR DESCRIPTION
**Pull request by past**
This captures my understanding of the meeting consensus after today's [discussion](https://github.com/web-platform-tests/wpt-notes/blob/master/minutes/2024-06-04.md#rfc-process-review). I didn't try to cover everything that was discussed, just tried to document the policy updates that would avoid a situation like #182 in the future.

We could make more updates, e.g. around what happens if the infra meeting discussion doesn't result in PR comments, what should be the relevant timeout. Another question we might want to tackle is documenting the policy for reverting (#187), but I thought I would start with what I didn't hear a clear objection on. 

I'll bring this up in the next WPT infra meeting, too.

### CODE REVIEW

**Comment by past** [comment]
Forgot to tag people for comments: @jgraham, @gsnedders, @foolip, @jcscottiii, @web-platform-tests/admins.

File: README.md [hunk 1]
```diff
@@ -37,9 +37,15 @@ Cases where the RFC process need *not* be used:
...
    proposal will change in accordance with that discussion.
- - In the case of no substantive disagreement the RFC is considered accepted
-   after 1 week. If any participant requests it, the comment period is extended
-   to 2 weeks.
+ - At least one review approval is required to merge a PR. For changes that
+   may impact downstream WPT consumers more approvals should be sought, or at
+   least a clear acknowledgement from those consumers that this change can be
+   accommodated in their workflows.
+ - If reviewers or downstream consumers are not responsive after at least two
+   weeks, the PR should be added to the monthly [WPT infra meeting](https://github.com/web-platform-tests/wpt-notes) to ensure it gets discussed. Anyone is welcome to file issues in that repo to request the discussion of an RFC.
```
Comment (past) [inline README.md:45]: Another option besides filing issues is tagging RFCs with an `agenda+` label, like we do in the WHATWG repos.

File: README.md [hunk 2]
```diff
@@ -37,9 +37,15 @@ Cases where the RFC process need *not* be used:
...
  - The proposal is discussed by the community and it is assumed that the
    proposal will change in accordance with that discussion.
- - In the case of no substantive disagreement the RFC is considered accepted
-   after 1 week. If any participant requests it, the comment period is extended
-   to 2 weeks.
+ - At least one review approval is required to merge a PR. For changes that
+   may impact downstream WPT consumers more approvals should be sought, or at
+   least a clear acknowledgement from those consumers that this change can be
+   accommodated in their workflows.
+ - If reviewers or downstream consumers are not responsive after at least two
```
Comment (foolip) [inline README.md:44]: Does one need to wait two weeks, or is it OK to put it on the agenda for the next WPT infra meeting when the RFC is filed?

Comment (past) [reply README.md:44]: No need to wait, happy to reword this to make it clearer.

File: README.md (hunk 1 above)
Comment (foolip) [reply README.md:45]: We do that in the interop team too now, so I like this option.

File: README.md (hunk 2 above)
Comment (foolip) [reply README.md:44]: Sounds good! Something like feel free to add it to the agenda any time you see fit, but in particular if nobody is responding to the RFC.

File: README.md (hunk 1 above)
Comment (past) [reply README.md]: I prefer it too, I updated the RFC accordingly.

File: README.md (hunk 2 above)
Comment (past) [reply README.md:48]: Thanks, I reworded this section to capture your suggestion.

**Review by Ms2ger — APPROVED** [review]
wfm

**Review by jcscottiii — APPROVED** [review]

**Comment by past** [comment]
@jgraham did you have any further comments here or should I merge this?

File: README.md [hunk 3]
```diff
@@ -37,9 +37,21 @@ Cases where the RFC process need *not* be used:
     * Risks
  - The proposal is discussed by the community and it is assumed that the
    proposal will change in accordance with that discussion.
- - In the case of no substantive disagreement the RFC is considered accepted
-   after 1 week. If any participant requests it, the comment period is extended
-   to 2 weeks.
+ - At least one review approval is required to merge a PR. For changes that
+   may impact downstream WPT consumers more approvals should be sought, or at
+   least a clear acknowledgement from those consumers that this change can be
+   accommodated in their workflows.
```
Comment (jgraham) [inline README.md:43]:
```suggestion
 - At least one review approval is required to merge a PR. For changes that
   may impact downstream WPT consumers there is an additional requirement
   of agreement from those consumers that the proposed change can be
   accommodated in their workflows.
```

File: README.md [hunk 4]
```diff
@@ -37,9 +37,21 @@ Cases where the RFC process need *not* be used:
...
+   [WPT infra meeting](https://github.com/web-platform-tests/wpt-notes) to
+   have a dsicsussion about it. To add an RFC to the agenda, tag the PR with the
+   `agenda+` label.
+ - If reviewers or downstream consumers are not responsive after at least two
+   weeks, the PR should be added to the monthly
+   [WPT infra meeting](https://github.com/web-platform-tests/wpt-notes) to
+   ensure it gets discussed.
+ - After review approval(s) and in the case of no substantive disagreement, the
+   RFC is considered accepted 1 week after the first approval. If any participant
+   requests it, the comment period is extended to 2 weeks.
```
Comment (jgraham) [inline README.md:54]:
```suggestion
 - After review approval(s) and in the case of no substantive disagreement, the
   RFC is considered accepted 1 week after the first approval. If any participant
   requests it, more time can be granted to review the proposal. This will not be
   less than one week but may be longer (e.g. until the next WPT infra meeting).
   However if no feedback is forthcoming by the agreed date, the proposal may
   be considered accepted.
```
